spec/index.html

```diff
@@ -358,6 +358,21 @@ <h2>Canonicalization Algorithm Terms</h2>
       <dt><dfn>normalized dataset</dfn></dt>
       <dd>The immutable, abstract <a>RDF dataset</a> and set of normalized
         <a>blank node identifiers</a> that are produced as output by the algorithm. A <a>normalized dataset</a> is a restriction on an <a>RDF dataset</a> where all nodes are labeled, and <a>blank nodes</a> are labeled with <a>blank node identifiers</a> consistent with running this algorithm on a base <a>RDF dataset</a>. A concrete serialization of an <a>normalized dataset</a> MUST label all <a>blank nodes</a> using these stable <a>blank node identifiers</a>.</dd>
+      <dt><dfn data-lt="canonical n-quads">canonical n-quads form</dfn></dt>
+      <dd>
+        The canonicalized representation of a quad is based on
+        <a data-cite="N-Triples#canonical-ntriples">canonical N-Triples</a>
+        defined in [[N-Triples]].
+        A represent a <a>graph name</a> in the same manner as a <a>subject</a>,
```

Comment:
Some words might as well be inserted between "A" and "represent", if I am not mistaken:
```suggestion
        A quad in <a>canonical n-quads form</a> represents a <a>graph name</a> in the same manner as a <a>subject</a>,
```